packages/wonder-blocks-dropdown/components/single-select.js

```diff
@@ -105,6 +116,12 @@ type State = {|
      * Whether or not last opened state change was triggered by a keyboard click.
      */
     keyboard?: boolean,
+
+    /**
+     * The text input to filter the items by their label. Defaults to an empty
+     * string.
+     */
+    searchText: string,
```

Comment:
Is this something that we'll want to have on `MultiSelect` and `ActionMenu` as well?  If so, it might make sense to push some of the logic you added to this component into dropdown-core.js.

Reply:
I see now that `searchText` is already part of `DropdownCore`.